docs/DeveloperGuide.md

```diff
@@ -292,6 +292,55 @@ Aspect: What fields should the `find` command search through?
   - Pros: Easy to implement, with minimal modification to existing tests.
   - Cons: Limited breadth of search, does not align with definition of unique application. 
 
+### Sort Feature
+
+#### Implementation
+
+The sort feature allows the user to sort the application list using company names, positions, application dates or interview dates. Each of these orders can also be reversed.
```

Comment:
Does the current implementation allow the user to sort Interview?

Reply:
Not yet, I'll quickly add that in next iteration